src/Altinn.Apps/AppFrontend/react/receipt/tsconfig.json

```diff
@@ -16,7 +22,7 @@
     "allowJs": true,
     "jsx": "react",
     "moduleResolution": "node",
-    "rootDir": "../",
+    "rootDir": "./../",
```

Comment:
Does this change make any sense?

Reply:
no, this should not be needed. Reverted this change.